P1684-mdarray/P1684.md

```diff
@@ -334,6 +342,397 @@ In R3 we decided to radically cut back on constructors and simply leave those ou
 
 Finally, `mdarray` does not have the analog of `mdspan`'s constructor that takes an `array<IndexType, N>` of dynamic extents.  This avoids any ambiguity or confusion with a constructor that takes a container instance, in the case where the `container` happens to be an `array<IndexType, N>`.
 
+### Alternative constructor design
```

Comment:
Is this the "alternative" design (meaning the "Plan B" that we would accept in case LEWG rejected Plan A), or the design that we want LEWG to approve?  If the latter, should we consider not using the word "alternative" here?  For example, should we call this "New R5 constructor design"?

Reply:
Well, it's the one **I** want LEWG to approve. :-)

I'll change the phrasing.  I want to emphasize that this hasn't been seen yet.